Dockerfile

```diff
@@ -52,9 +55,17 @@ RUN apt-get update && \
         && \
     rm -rf /var/lib/apt/lists/*
 
+COPY --from=builder \
+     mbsolr-jaxb/target/mbsolr-jaxb-1.0-SNAPSHOT-jar-with-dependencies.jar \
+     /opt/solr/server/lib/
+
 COPY --from=builder \
      mb-solr/target/mb-solr-0.0.1-SNAPSHOT-jar-with-dependencies.jar \
-     /opt/solr/lib/
+     /opt/solr/server/solr/lib/
+
+ENV SOLR_HOME /opt/solr-9.0.0/server/solr/
+COPY --chown=solr:solr ./mbsssss $SOLR_HOME/mycores/mbsssss
```

Comment:
This seems to be a bug due to my merge when making this PR, there is another COPY a few lines lower. I will change it.